runtime/moonbeam/src/governance/democracy.rs

```diff
@@ -35,33 +35,33 @@ impl pallet_democracy::Config for Runtime {
 	type MinimumDeposit = ConstU128<{ 4 * currency::GLMR * currency::SUPPLY_FACTOR }>;
 	/// To decide what their next motion is.
 	type ExternalOrigin =
-		pallet_collective::EnsureProportionAtLeast<AccountId, CouncilInstance, 1, 2>;
```

Comment:
Is it fine for the OpenTechComitee to replace the Council here or should it be another open gov custom track/origin?

Reply:
Democracy will be remove, and we don't whant to create a new track/origin for a deprecatde pallet, so in the meantime we are using the open tech comittee